innovation.game.php

```diff
@@ -16034,6 +16038,7 @@ function stInterInteractionStep() {
                     // Put all cards back in hand
                     $cards = self::getCardsInLocation($player_id, 'revealed');
                     foreach ($cards as $card) {
+                        $card = self::getCardInfo($card['id']);
```

Comment:
This bug doesn't actually happen when the card is in a publicly visible location (e.g. revealed or board). So let's remove this line so that we aren't making an unnecessary SQL query.